package.json

```diff
@@ -45,6 +45,7 @@
     "react-native-screens": "^2.4.0",
     "react-native-snackbar": "^2.2.3",
     "react-native-svg": "^12.1.0",
+    "react-native-svg-uri": "^1.2.3",
```

Comment:
Since react-native 0.6x native npm modules are using auto-linking and (mostly) does not need to get manually linked any more. So please remember this for adding future modules:
```
error React Native CLI uses autolinking for native dependencies, but the following modules are linked manually: 
  - react-native-svg (to unlink run: "react-native unlink react-native-svg")
```
I will try to unlink the module now and remove the unnecessary code which cannot be removed by unlinking manually.

Reply:
Ok. Thanks